packages/appsync-emulator-serverless/bin/server.js

```diff
@@ -45,8 +45,7 @@ const main = async () => {
   });
   // argparse converts any argument with a dash to underscores
   // eslint-disable-next-line
-  let { wsport, port, path: serverlessPath, dynamodb_port: dynamodbPort } = parser.parseArgs();
-
+  let { ws_port, port, path: serverlessPath, dynamodb_port: dynamodbPort } = parser.parseArgs();
```

Comment:
nitpick: I think something like:

```
ws_port: wsPort,
```

will allow us to keep camel casing the variables for consistency